go.mod

```diff
@@ -75,3 +75,5 @@ require (
 )
 
 go 1.13
+
+replace github.com/pingcap/parser => github.com/crazycs520/parser v0.0.0-20191218032222-cb5d742ae9e0
```

Comment:
。。。

Reply:
https://github.com/pingcap/parser/pull/690 merge this first